Programs/Asotin_IMW/Program.xml

```diff
@@ -0,0 +1,160 @@
+        <Allow id="Upper Salmon" name="Upper Salmon" folder="UpperSalmon" />
+        <Allow id="Upper Selway" name="Upper Selway" folder="UpperSelway" />
+        <Allow id="Upper Yakima" name="Upper Yakima" folder="UpperYakima" />
+        <Allow id="Walla Walla" name="Walla Walla" folder="WallaWalla" aliases="WallaWalla" />
+        <Allow id="Wallowa" name="Wallowa" folder="Wallowa" />
+        <Allow id="Wenatchee" name="Wenatchee" folder="Wenatchee" />
+        <Allow id="Willow" name="Willow" folder="Willow" />
+        <Allow id="Yankee Fork" name="Yankee Fork" folder="YankeeFork" aliases="YankeeFork" />
+      </Collection>
+      <Collection id="COL_SITE" name="Site">
+        <AllowRegex pattern=".*" />
+      </Collection>
+      <Collection id="COL_VISIT" name="Visit">
+        <AllowRegex pattern="^[0-9]{2,6}$" />
+      </Collection>
+      <Collection id="COL_YEAR" name="Year">
+        <AllowRegex pattern="^[0-9]{4}$" />
+      </Collection>
+      <Collection id="COL_FLOW" name="Flow">
+        <AllowRegex pattern=".*" />
+      </Collection>
+    </Collections>
+
+    <!--Groups are a lot simpler than collections. They can only be a single folder-->
+    <Groups>
+      <Group id="GRP_BASE" name="CHaMP" folder="CHaMP" />
+      <Group id="GRP_WATERSHED" name="Watershed" folder="Watershed" />
+      <Group id="GRP_NETWORK" name="Network" folder="Network" />
+      <Group id="GRP_SITE" name="Site" folder="Site" />
+      <Group id="GRP_YEAR" name="Year" folder="Year" />
+      <Group id="GRP_FLOW" name="Flow" folder="Flow" />
+      <Group id="GRP_TOPO" name="Topography" folder="Topography" />
+    </Groups>
+
+    <!--Products for now just need ref, name and folder (url is optional and leads to a wiki)-->
+    <Products>
+      <!--NETWORK level products-->
+      <Product id="CAD_Export" name="CAD Export" folder="CAD_Export" url="" />
+
+      <!--Site-level products-->
+      <Product id="GCD" name="GCD" folder="GCD" url="" />
+
+      <!--Visit-level products-->
+      <Product id="Topo" name="Topo" folder="Topo" url="" />
+
+      <!--Flow-level products-->
+      <Product id="FHM" name="FHM" folder="FHM" url="" />
+      <Product id="GUT" name="GUT" folder="GUT" url="" />
+      <Product id="Hydro" name="Hydro" folder="Hydro" url="" />
+
+      <!--HUC8 Level Projects -->
+      <Product id="BRAT" name="BRAT Models (HUC8)" folder="BRAT" url="" />
+      <Product id="RSContext" name="Riverscapes Context (HUC8)" folder="RSContext" url="" />
+
+    </Products>
+  </Definitions>
+
+  <!-- Here's where we specify the actual structure
+  Reminder:
+  - Collection: Repeating folders (e.g. Asotin, Lemhi
+  - Group: Single folder with an explicit name (e.g. Topography)
+  Rules:
+  - Only one collection is allowed at any level
+  - Collections cannot be mixed with groups or products
+-->
+  <Hierarchy>
+    <Group ref="GRP_BASE">
+      <Group ref="GRP_WATERSHED">
+        <Collection ref="COL_WATERSHED">
```

Comment:
If this is all Asotin do we need the watershed group and collection at all?

Reply:
Let's keep watershed so we can integrate back into CHaMP if needed.